Lib/test/test_ssl.py

```diff
@@ -33,6 +34,13 @@
 def data_file(*name):
     return os.path.join(os.path.dirname(__file__), *name)
 
+def _can_decode_unicode(unicode_str):
```

Comment:
Since the function is only used once, I would prefer to see it inlined in the test.